.github/workflows/cmd.yml

```diff
@@ -0,0 +1,417 @@
+
+      - name: Build workflow link
+        if: ${{ !contains(github.event.comment.body, '--quiet') }}
+        id: build-link
+        run: |
+          # Get exactly the CMD job link, filtering out the other jobs
+          jobLink=$(curl -s \
+            -H "Authorization: token ${{ secrets.GITHUB_TOKEN }}" \
+            -H "Accept: application/vnd.github.v3+json" \
+            https://api.github.com/repos/${{ github.repository }}/actions/runs/${{ github.run_id }}/jobs | jq '.jobs[] | select(.name | contains("${{ env.JOB_NAME }}")) | .html_url')
+          
+          runLink=$(curl -s \
+            -H "Authorization: token ${{ secrets.GITHUB_TOKEN }}" \
+            -H "Accept: application/vnd.github.v3+json" \
+            https://api.github.com/repos/${{ github.repository }}/actions/runs/${{ github.run_id }} | jq '.html_url')
+          
+          echo "job_url=${jobLink}"
+          echo "run_url=${runLink}"
+          echo "job_url=$jobLink" >> $GITHUB_OUTPUT
+          echo "run_url=$runLink" >> $GITHUB_OUTPUT
+
+
+      - name: Comment PR (Start)
+        if: ${{ !contains(github.event.comment.body, '--quiet') }}
+        uses: actions/github-script@v7
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          script: |
+            let job_url = ${{ steps.build-link.outputs.job_url }}
+            
+            github.rest.issues.createComment({
+              issue_number: context.issue.number,
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              body: `Command "${{ steps.get-pr-comment.outputs.group2 }}" has started 🚀 [See logs here](${job_url})`
+            })
+
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ github.head_ref }}
+
+      - name: Install dependencies for bench
+        if: startsWith(steps.get-pr-comment.outputs.group2, 'bench')
+        run: cargo install subweight frame-omni-bencher --locked
+
+      - name: Run cmd
+        id: cmd
+        env:
+          CMD: ${{ steps.get-pr-comment.outputs.group2 }} # to avoid "" around the command
+        run: |
+          echo "Running command: '$CMD' on '${{ needs.set-image.outputs.RUNNER }}' runner, container: '${{ needs.set-image.outputs.IMAGE }}'"
+          echo "RUST_NIGHTLY_VERSION: $RUST_NIGHTLY_VERSION"
+          # Fixes "detected dubious ownership" error in the ci
+          git config --global --add safe.directory '*'
+          git remote -v
+          python3 .github/scripts/cmd/cmd.py $CMD
+          git status
+          git diff
+
+      - name: Commit changes
+        run: |
+          if [ -n "$(git status --porcelain)" ]; then
+            git config --local user.email "action@github.com"
+            git config --local user.name "GitHub Action"
+          
+            git pull origin ${{ github.head_ref }}
+            git add .
+            git restore --staged Cargo.lock # ignore changes in Cargo.lock
```

Comment:
There should be none. Maybe it was because of the `cargo metadata`. If there are changes to the Cargo.lock then it means that we benchmarked something different from what we should have.

Reply:
This part is ported from old command bot, perhaps there was other use-cases, but i think it's better to have this set explicitly, to avoid accidental (silent) change of lock